### PR TITLE
Document the spec for refunds & changes to invoiced orders

### PR DESCRIPTION
**Pull request by gldraphael**
Related #230, 3a3f325, e86dd3b, #225, #208, #181

### CODE REVIEW

File: docs/Specification/Orders.md [hunk 1]
```diff
@@ -18,5 +18,116 @@ After implementing orders, we should change from verification of registration to
...
+5 × C
+1 × D
+```
+
+Case 2: Removing an item from Order #255.
+
+```text
+Order #256
+------------------------
+R3  Item-C  ×5   (Kr500)
```
Comment (losviko) [inline docs/Specification/Orders.md]: note that the quantity would be -5

**Comment by gldraphael** [comment]
Here's an edge case considering the fact that the price of a orderline can be edited (before its order is invoiced):

```
Product     Price
A	    1000kr
B   	    1000kr
```

```
Order #311
A	x1	500kr
B	x1	500kr
```

```
Order #312
A	x1	1000kr
```

The products are now:
```
A x2
B x2
```
How do we change the quantity of `A` to `1`.
What should Order #313 look like?

**Comment by losviko** [comment]
Excellent question... 
Order #313 should take the price of the product at the moment, but ideally give a notice to the admin to check which price to refund..

**Comment by gldraphael** [comment]
1. The spec only covers cases where existing orders are invoiced. Need to work on cases where non-invoiced existing orders are considered. 
1. The current spec doesn't handle the change of product variants well. Two possible outcomes can be:
    1. Just add the new variant and leave the existing variant.
    1. Replace the existing variant with the new variant.

**Pull request merged**